order/buyer/indicators.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Where is this page listed in Gitbook? I can't find it in the menu..

Reply:
Its linked from issue.md and will be from other use case pages that use indicators